playground/src/utils/http-method-string.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Returns a string representation of the HTTP method.
+ *
+ * In practice, shortens the method name to be 3-5 characters.
+ */
+export function getHttpMethodString(method: string) {
```

Comment:
added this to never have more than 5 chars in the Method component, or when we show an HTTP method elsewhere